### PR TITLE
Add libnss3-tools installation to chrome and edge image

### PR DESCRIPTION
**Pull request by algoritmus**
**Thanks for contributing to the Docker-Selenium project!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines, applied for this repository.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
Added libnss3-tools package to image in order to be able to add custom certificates, WAs, etc to nssdb, or be able to create new nssdb. Nssdb certificate storage is used by chromium based browsers (Chrome, Edge)

### Motivation and Context
In order to be able to add custom certificates, WAs, etc to nssdb, or be able to create new nssdb. Nssdb certificate storage is used by chromium based browsers (Chrome, Edge)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by diemol — CHANGES_REQUESTED** [review]
Changes to Nodes should be applied to this file https://github.com/SeleniumHQ/docker-selenium/blob/trunk/NodeBase/Dockerfile.txt

**Comment by algoritmus** [comment]
Corrected

**Review by diemol — CHANGES_REQUESTED** [review]
Still missing docs showing how to use the feature. Could you please add them to the README?

File: README.md [hunk 1]
```diff
@@ -822,6 +823,31 @@ Like this, the script will poll until the Grid is ready, and then your tests wil
...
+## Install cerificates
+
+If you need to install custom certificates, CA, intermediate CA, client certificates (for exmample enterprise internaöl CA)
+you can use the different utils come from libnss3-tools.
+
+Chromium based browser uses nssdb as certificate store
+
+Create new nssdb:  
+```bash
+certutil -d sql:$HOME/.pki/nssdb -N
```
Comment (diemol) [inline README.md:835]: But how would one execute those commands inside the container?

Can you please add a more complete example? I am sure issues will be created asking that.

**Review by diemol — CHANGES_REQUESTED** [review]

**Comment by algoritmus** [comment]
Added usage to readme

File: README.md (hunk 1 above)
Comment (algoritmus) [reply README.md:835]: readme extended with usage

File: README.md [hunk 2]
```diff
@@ -822,6 +823,53 @@ Like this, the script will poll until the Grid is ready, and then your tests wil
...
+RUN mkdir -p -m755 /home/seluser/.pki/nssdb \ #create nssdb folder
+    && certutil -d sql:/home/seluser/.pki/nssdb -N --empty-password \ # create new db without password
+    && certutil -d sql:/home/seluser/.pki/nssdb -A -t "C,," -n companyca -i /etc/certs/companeca.pem \ #trust company CA
+    && pk12util -d sql:/home/seluser/.pki/nssdb -i client_cert.p12 -W password_of_clent_cert # client certificate install
+
+```
+This way the certificates will be installed and the node will start automatically as before.
+
+___
+
```
Comment (diemol) [inline README.md:872]: Apologies if I am being too exhaustive in the review, but if a user needs to create their own image to install the certificates, what is the benefit of installing the tools beforehand?

This PR could be only adding this example to the documentation, and showing how the tools can be installed.

Am I missing something?

Comment (diemol) [reply README.md:872]: If you want to have a quicker conversation, feel free to drop by our chat [(Matrix, IRC or Slack)](https://www.selenium.dev/support/#ChatRoom), and we can get this done.

**Comment by algoritmus** [comment]
The point would be that usually companies allows no access to external
repos, so cannot do the install part of the libnss3. It is usually allows
to get docker images but cannot do any apt-get. Ok, frankly speaking here I
am a bit selfish, and this would help me, rather create an image at home
with libnss install then use that image to add certs within enterprise
environment.

Feel free to reject the PR if you think it would serve too narrow user base.

Attila

**Comment by diemol** [comment]
@algoritmus thanks for being open about it. I am not sure if it is a common use case or not.

These images try to have as few things as possible to cover most use cases. However, I know some people ask every now and then about installing certificates. If you want, I'd be happy to receive a contribution to the README showing how to do it. Would you be OK with that?

**Comment by algoritmus** [comment]
Sure, no problem.

**Pull request closed without merging**